lib/trento/domain/host/host.ex

```diff
@@ -242,6 +249,24 @@ defmodule Trento.Domain.Host do
     }
   end
 
+  # Start the rollup flow
+  def execute(
+        %Host{host_id: nil},
+        %RollUpHost{}
+      ) do
+    {:error, :host_not_registered}
+  end
+
+  def execute(
+        %Host{host_id: host_id} = snapshot,
+        %RollUpHost{}
+      ) do
+    %HostRollUpRequested{
+      snapshot: snapshot,
+      host_id: host_id
```

Comment:
Maybe switch the order of the fields to be consistent with the event fields order